SU-SOE/engineering_profile
# SOED9-8 - added permissions for editors and contributors.

# READY FOR REVIEW

# Summary
- allows editors and contributors limit access to admin for content editing and authoring. 

# Review By (Date)
-  ASAP

# Criticality
- 10

# Urgency
- High

## Steps for testing

You can enable the masquerade module and test to make sure uses have the correct access.
